src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationService.cs

```diff
@@ -109,6 +109,10 @@ internal class CSharpRecommendationService : AbstractRecommendationService
             {
                 return GetSymbolsOffOfExpression(context, ((MemberAccessExpressionSyntax)node).Expression, cancellationToken);
             }
+            else if (node.Kind() == SyntaxKind.RangeExpression)
+            {
+                return GetSymbolsOffOfExpression(context, ((RangeExpressionSyntax)node).LeftOperand, cancellationToken);
```

Comment:
probably worth it to have a comment stating you only get here when the user is after the first dot of the range operator.